jans-cedarling/cedarling/src/jwt/decoding_strategy.rs

```diff
@@ -142,24 +120,13 @@ fn decode_and_validate_jwt<T: DeserializeOwned>(
 
     // set up validation rules
     let mut validator = jwt::Validation::new(header.alg);
-    let mut required_spec_claims = vec!["iss", "sub", "aud"];
-    if decoding_args.validate_exp {
-        required_spec_claims.push("exp");
-    }
-    validator.set_required_spec_claims(&required_spec_claims);
-    validator.validate_nbf = decoding_args.validate_nbf;
-    validator.validate_exp = decoding_args.validate_exp;
-    if let Some(iss) = decoding_args.iss {
-        validator.set_issuer(&[iss]);
-    } else {
-        validator.iss = None;
-    }
-    if let Some(aud) = decoding_args.aud {
-        validator.set_audience(&[aud]);
-    } else {
-        validator.validate_aud = false;
-    }
-    validator.sub = decoding_args.sub.map(|sub| sub.to_string());
+    // We clear the required claims because the validator requires
+    // `exp` by default.
+    validator.required_spec_claims.clear();
+    // `aud` should be optional.
+    validator.validate_aud = false;
```

Comment:
I think these flags will decide if the claims are optional or mandatory. Are we setting the default value here?
And user can reset these values in the bootstrap configuration.

Reply:
this is just a quick fix, so we can test tokens for MVP since i still haven't finished #10142.